spec/koans.js

```diff
@@ -408,46 +412,48 @@ describe('destructuring function parameters. ', () => {
   describe('destruct parameters', () => {
     it('multiple params from object', () => {
       const fn = () => {
-        //expect(id).toEqual(42);
-        //expect(name).toEqual('Wolfram');
+        expect(id).toEqual(42);
+        expect(name).toEqual('Wolfram');
       };
       const user = {name: 'Wolfram', id: 42};
+      let {id, name} = user;
       fn(user);
```

Comment:
Haha, guys, this is not a race !! or if it is one, you won it, you faster at going on with the code than me correcting it :)
anyway, here the point is to understand that you can also destrucure parameters , and it works like this : 
```js

    it('multiple params from object', () => {
      const fn = ({id, name}) => {
        expect(id).toEqual(42);
        expect(name).toEqual('Wolfram');
      };
      const user = {name: 'Wolfram', id: 42};
      fn(user);
    });
```
This one is actually super useful in backend routes !